components/automate-ui/src/app/entities/automate-settings/automate-settings.requests.ts

```diff
@@ -22,49 +27,123 @@ export class AutomateSettingsRequests {
   // fetchJobSchedulerStatus sends an HTTP GET Request to read the status of the JobScheduler
   // living inside the ingest-service
   public fetchJobSchedulerStatus(_params): Observable<JobSchedulerStatus> {
-    const url = `${RETENTION_URL}/nodes/status`;
+
+    const url = `${RETENTION_URL}/status`;
 
     return this.http
       .get<RespJobSchedulerStatus>(url).pipe(
       map((res) => this.convertResponseToJobSchedulerStatus(res)));
   }
 
-  // configureIngestJob sends an HTTP POST Request to the provided ingest job to configure
+  // configureIngestJob sends an HTTP PUT Request to the provided ingest job to configure
   // it with the provided threshold and running state
-  configureIngestJob(job: IngestJob): Observable<any> {
-    let url: string;
+  configureIngestJobs(jobs: IngestJob[]): Observable<any> {
+    const url = `${RETENTION_URL}/config`;
 
-    switch (job.name) {
-      case IngestJobs.MissingNodes: {
-        url = `${RETENTION_URL}/nodes/missing-nodes/config`;
-        break;
+    const body: JobRequestBody = {
+      infra : {
+        job_settings: [
+          {
+            name: 'periodic_purge_timeseries',
+            purge_policies: {
+              elasticsearch: []
+            }
+          }
+        ]
+      },
+      compliance: {
+        job_settings: [
+          {
+            name: 'periodic_purge',
+            purge_policies: {
+            elasticsearch: []
+            }
+          }
+        ]
+      },
+      event_feed: {
+        job_settings: [
+          {
+            name: 'periodic_purge',
+            purge_policies: {
+            elasticsearch: []
+            }
+          }
+        ]
       }
-      case IngestJobs.MissingNodesForDeletion: {
-        url = `${RETENTION_URL}/nodes/missing-nodes-deletion/config`;
-        break;
-      }
-      case IngestJobs.DeleteNodes: {
-        url = `${RETENTION_URL}/nodes/delete-nodes/config`;
-        break;
+    };
+
+    jobs.forEach(job => {
+      let thisJob = new UnfurledJob;
+
+      switch (job.name) {
+        case InfraJobName.DeleteNodes:              // fallthrough
+        case InfraJobName.MissingNodes:             // fallthrough
+        case InfraJobName.MissingNodesForDeletion:
+          thisJob = this.unfurlIngestJob(job);
+          // Protect against an undefined value
+          if (body[job.category].job_settings) {
+            body[job.category].job_settings.push(thisJob);
+          } else {
+            console.log('TODO: send an error');
```

Comment:
Not sure if this is the right approach here - Open to opinions on how to protect against an undefined value.